app/tests/spec/views/connect_another_device.js

```diff
@@ -116,14 +139,20 @@ define(function (require, exports, module) {
           sinon.stub(user, 'isSignedInAccount', () => false);
           sinon.stub(view, '_canSignIn', () => true);
 
-          return view.render();
+          return view.render()
+            .then(() => {
+              view.afterVisible();
+            });
         });
 
         it('shows a sign in button with the appropriate link, logs appropriately', () => {
           assert.lengthOf(view.$('#signin'), 1);
           testIsNotified('connectAnotherDevice.signedin.false');
           testIsNotified('connectAnotherDevice.signin.eligible');
           testIsNotified('connectAnotherDevice.signin_from.fx_desktop');
+          testIsFlowEventLogged('signedin.false');
+          testIsFlowEventLogged('signin.eligible');
+          testIsFlowEventLogged('signin_from.fx_desktop');
```

Comment:
Those event docs are next in the line of fire on my auto-generate-all-the-docs mission.